locations/templates/locations/location_page.html

```diff
@@ -1,68 +1,241 @@
+                                                    <p>
+                                                        <b>
+                                                            Monday: {{ page.mon_hours|richtext }}<br>
+                                                            Tuesday: {{ page.tue_hours|richtext }}<br>
+                                                            Wednesday: {{ page.wed_hours|richtext }}<br>
+                                                            Thursday: {{ page.thu_hours|richtext }}<br>
+                                                            Friday: {{ page.fri_hours|richtext }}<br>
+                                                            Saturday: {{ page.sat_hours|richtext }}<br>
+                                                            Sunday: {{ page.sun_hours|richtext }}<br>
+                                                        </b>
+                                                    </p>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                {% endif %}
+                                <div class="page-body mt-4 body-font">
+                                    {{ page.body|richtext }}
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="placeholder-div"></div>
+                    <div class="shadow-lg mx-auto rounded" style="width: 100vw;">
+                        <div class="map" id="map"></div>
+                    </div>
                 </div>
-            </div>
-        {% endfor %}
-    </div>
 
-    {% if page.lat_long %}
-        <div class="row">
-            <div class="card shadow-lg p-3 mx-auto rounded">
-                <iframe
-                    class="mx-auto d-block"
-                    width="400" height="250" frameborder="0" style="border:0"
-                    src="https://www.google.com/maps/embed/v1/view?key=AIzaSyBKC9GWNbQ95x0CH2iQvOkTZdwoP3aEjYM&center={{page.specific.lat_long}}&zoom=18" allowfullscreen>
-                </iframe>
-            </div>
-        </div>
-    {% endif %}
-
-    {% if page.tags.all.count %}
-        <div class="row">
-            <div class="col m-3">
-                <div class="text-center">
-                    <h5>Tags</h5>
-                    {% for tag in page.tags.all %}
-                        <a href="{% slugurl 'tags' %}?tag={{ tag }}"><button type="button" class="btn-sm btn-info">{{ tag }}</button></a>
-                    {% endfor %}
+                <div class="mt-5 text-center">
+                    <a href="https://www.google.com/maps/@{{page.specific.lat_long}},18z"><button type="button" class="btn btn-outline-primary">Open in Google Maps</button></a>
```

Comment:
lets get rid of this google map link, or just make the link when you click the actual map image?